axis/resonant_axis.f95

```diff
@@ -0,0 +1,111 @@
+module resonant_axis
+
+use global_parameters
+implicit none
+
+contains
+
+!----------------------------------------------------------------------------------------------
+subroutine make_id_matrix_2body(pl_name,max_order)
```

Comment:
make -> build
This subroutine should have a unit test (that the file has been created & it is correct).